src/Http/Http/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+Microsoft.AspNetCore.Http.StreamResponseBodyFeature.StreamResponseBodyFeature(System.IO.Stream! stream, Microsoft.AspNetCore.Http.Features.IHttpResponseBodyFeature? priorFeature) -> void
```

Comment:
```suggestion
*REMOVED*Microsoft.AspNetCore.Http.StreamResponseBodyFeature.StreamResponseBodyFeature(System.IO.Stream! stream, Microsoft.AspNetCore.Http.Features.IHttpResponseBodyFeature! priorFeature) -> void
Microsoft.AspNetCore.Http.StreamResponseBodyFeature.StreamResponseBodyFeature(System.IO.Stream! stream, Microsoft.AspNetCore.Http.Features.IHttpResponseBodyFeature? priorFeature) -> void
```

Reply:
You beat me to it! @dotnet/aspnet-build Any idea why VS doesn't do it this way automatically? Is it not supposed to?

Reply:
I'm not sure, @agocke do you know who we should consult with on the API analyzer?

Reply:
The codefix updates the shipping file as intended. For some reason our team insists it is wrong thing to do: https://github.com/dotnet/aspnetcore/pull/31076#discussion_r598208196 even those these files exist solely for the benefit of the tool. 🤷🏽